staging/src/k8s.io/apiserver/pkg/server/httplog/httplog.go

```diff
@@ -223,8 +223,9 @@ func (rl *respLogger) Hijack() (net.Conn, *bufio.ReadWriter, error) {
 	return rl.w.(http.Hijacker).Hijack()
 }
 
-// CloseNotify implements http.CloseNotifier
+// CloseNotify implements http.CloseNotify.
 func (rl *respLogger) CloseNotify() <-chan bool {
+	//lint:ignore SA1019 dropping this method means this handler no longer implements that interface
 	return rl.w.(http.CloseNotifier).CloseNotify()
 }
```

Comment:
Can you add an interface implementation check to make sure we don't drop this method again? E.g.,
```golang
var _ http.CloseNotifier = &respLogger{}
```